wagtailio/core/blocks.py

```diff
@@ -424,11 +424,6 @@ class Meta:
 class LoopingVideoBlock(blocks.StructBlock):
```

Comment:
suggestion: rename this to `VideoBlock` to better reflect its nature. 
This would mean the homepage video will need re-adding, unless you do https://docs.wagtail.org/en/stable/advanced_topics/streamfield_migrations.html#streamfield-data-migrations

Also https://github.com/wagtail/wagtail.org/blob/b6f8ea896998ba6fb268ac7b3cb7fb8105da8e67/wagtailio/project_styleguide/templates/patterns/components/streamfields/looping_video_block/looping_video_block.html#L9 needs a tidy up.

Reply:
Good idea. I think re-adding the homepage video would be the preferred approach for managing this. If it's just one video, we can definitely do that. Will test it locally first of course.